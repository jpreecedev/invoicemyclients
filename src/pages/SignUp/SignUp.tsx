import React from 'react'

const SignUpPage = () => <p>Sign Up Form</p>

export { SignUpPage }
