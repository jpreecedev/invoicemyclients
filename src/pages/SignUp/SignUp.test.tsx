import React from 'react'
import { render } from '@testing-library/react'
import { SignUpPage } from './SignUp'

describe('<SignUp /> tests', () => {
  it('should match snapshot', () => {
    expect(render(<SignUpPage />).container).toMatchSnapshot()
  })
})
