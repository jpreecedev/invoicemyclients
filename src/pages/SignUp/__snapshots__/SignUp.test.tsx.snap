// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUp /> tests should match snapshot 1`] = `
<div>
  <section>
    <div
      class="container d-flex flex-column"
    >
      <div
        class="row align-items-center justify-content-center no-gutters min-vh-100"
      >
        <div
          class="col-12 col-md-6 py-8 py-md-11 mb-10"
        >
          <h1
            class="mb-0 font-weight-bold text-center"
          >
            Sign In or Register
          </h1>
          <p
            class="mb-6 text-center text-muted"
          >
            To ensure maximum security and simplicity, we recommend signing up/logging in using a secure authentication provider.
          </p>
          <div
            class="mb-8"
          />
          <small
            class="d-block text-center text-muted"
          >
            If you do not already have an account, we will create one for you.
          </small>
        </div>
      </div>
    </div>
  </section>
</div>
`;
